README.md

```diff
@@ -48,13 +48,13 @@ return [
     /*
      * The private key used to sign all URLs.
      */
-    'private_key_path' => storage_path(env('CLOUDFRONT_PRIVATE_KEY_PATH', 'trusted-signer.pem')),
```

Comment:
I avoided adding `AWS_` because I wanted to avoid possible future naming conflicts.
But I'm ok to change it if you think it's more easy to use.

Reply:
Thats a valid reason, I wasn't sure if it was intentional or not. I'll revert back to the original variables.